test/metabase/lib/field_test.cljc

```diff
@@ -0,0 +1,34 @@
+(ns metabase.lib.field-test
+  (:require
+   [clojure.test :as t]
```

Comment:
Is this deliberately a preferred style? `[clojure.test :refer [deftest is testing]]` etc. works in CLJS.

Reply:
I didn't realize you could do that TBH. I was pretty sure that didn't work at some point, right? I'll change it